libs/ui/src/molecules/Footer/Footer.tsx

```diff
@@ -0,0 +1,112 @@
+
+import Image from "next/image";
+import { FaTwitter, FaDiscord, FaTelegram } from "react-icons/fa";
+
+const logo = require('../../assets/images/logo.png');
+
+const Footer = () => {
+  return (
+    <footer style={{ background: `rgb(242,55,62) linear-gradient(179deg, rgba(242,55,62,1) 0%, rgba(33,54,174,1) 99%, rgba(34,54,175,1) 100%)` }}>
+      <div className="mx-auto max-w-7xl">
+        {/* First layer */}
+        <div className="relative flex h-44 pt-12">
+            <div className="items-center px-2 lg:px-0">
+              <Image
+                className="hidden h-8 w-auto lg:block"
+                src={logo}
+                alt="Your Company"
+              />
+              <div className="pt-4">
+                <div className="rounded-md text-sm font-medium text-white">
+                    Discover, collect, and sell digital items on the
+                </div>
+                <div className="rounded-md text-sm font-medium text-white">
+                  largest NFT marketplace on Optimism
+                </div>
+              </div>
+            </div>
+            <div className="flex flex-1 justify-center lg:justify-end">
+              <div className="lg:mr-20">
+                <h3 className="text-white text-xl font-medium">Join the community</h3>
+                <div className="flex pt-6 space-x-6">
+                  <button
+                      type="button"
+                      className="flex-shrink-0 px-2 py-2 rounded-lg bg-sky-900"
+                    >
+                  <FaTwitter className="h-6 w-6 text-white"/>
+                  </button>
+                  <button
+                      type="button"
+                      className="flex-shrink-0 px-2 py-2 rounded-lg bg-sky-900"
+                    >
+                  <FaDiscord className="h-6 w-6 text-white"/>
+                  </button>
+                  <button
+                      type="button"
+                      className="flex-shrink-0 px-2 py-2 rounded-lg bg-sky-900"
+                    >
+                  <FaTelegram className="h-6 w-6 text-white"/>
```

Comment:
it looks like the design has more padding / larger button total size (50px). could you adjust the sizes a little? also check the background color